language/stdlib/modules/libra_version.move

```diff
@@ -9,10 +9,13 @@ module LibraVersion {
     }
 
     public fun initialize() {
+        Transaction::assert(Transaction::sender() == LibraConfig::default_config_address(), 1);
 
-        LibraConfig::publish_new_config<Self::T>(T {
+        let cap = LibraConfig::publish_new_config<Self::T>(T {
```

Comment:
You shouldn't need to instantiate `publish_new_config` or `drop_capability` below; the compiler should be able to infer this information for you :)